Releases.md

```diff
@@ -1489,7 +1516,7 @@ Changes in std version 0.85.0:
 - fix: use inline source maps when present in js (#8995)
 - fix: use tokio for async fs ops (#9042)
 - refactor(cli): remove 'js' module, simplify compiler snapshot (#9020)
-- refactor(op_crates/crypto): Prefix ops with "op_crypto_" (#9067)
+- refactor(op*crates/crypto): Prefix ops with "op_crypto*" (#9067)
```

Comment:
Is this intentional? The other escapes look good. We should build that into the script that updates this file.

Reply:
I think Prettier ran when I saved and I didn't actually look at the diff 🙃